packages/vm/Cargo.toml

```diff
@@ -30,8 +30,6 @@ iterator = ["cosmwasm-std/iterator"]
 staking = ["cosmwasm-std/staking"]
 # this enables all stargate-related functionality, including the ibc entry points
 stargate = ["cosmwasm-std/stargate"]
-# Use cranelift backend instead of singlepass
-cranelift = ["wasmer/cranelift"]
```

Comment:
I guess this is technically breaking? Do we need to remove this to get the CI speedup?

Reply:
Otherwise we would have to manually list all the features across the entire workspace in the CI configuration. That feels pretty error-prone, making the coverage even more unreliable than it is in its current configuration.

Also yeah, technically it's a breaking change but realistically, who is depending on this feature? I don't think anyone is directly depending on the VM package (outside on libwasmvm), and even if, they'd also need to have `cranelift` enabled.

That's a pretty rare configuration to happen, to be honest.

I mean, we could also re-add the feature and make it a no-op, with a reference to remove it next breaking change.

Reply:
It could potentially also break users that use `cosmwasm-vm/cranelift` in their contract projects for testing the Wasm (same setup that you adjust here in this PR). However, hardly anyone does this these days because cosmwasm-vm based local tests are inconvenient and do not provide a good amount of additional insights.

Personally I'd be happy to get rid of the feature as the ARM and Windows compatibility of Singlepass made it unnecessary for all cases I am aware of. But no strong opinion on the migration path.

Reply:
Added back a no-op feature that enables nothing, that will keep existing setups like the ones you mentioned from breaking.

Reply:
Sounds good! I created #2262 for completely removing the feature in 3.0